bin/list-trello-board-lists.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# Load in our dependencies
+import argparse
+
+from trello import Board, TrelloClient
+
+
+# Define our logic
+def main(api_key, token, board_id):
+    """List out the board lists for our client"""
+    trello_client = TrelloClient(
+        api_key=api_key,
+        token=token,
+    )
+    board = Board(client=trello_client, board_id=board_id)
+    print('Lists')
```

Comment:
You really should do `from __future__ import print_function`.

Although this works, it is not for the reasons you think. This is actually printing a tuple of a single item, which python optimizes out to be a single element.

```python
print('hello')
# hello
print('hello', )
# ('hello', )
print('hello', 'world')
# ('hello', 'world')
```

Where in python3:

```python
print('hello')
# hello
print('hello', )
# hello
print('hello', 'world')
# hello world
```

Also, if you are worried, `__future__.print_function` is available in python, so you do not need to do a conditional import.